cron/activation-reminder-mail.py

```diff
@@ -0,0 +1,65 @@
+import time
+import os, sys
+from datetime import datetime, date, timedelta
+from django.db.models import Q
+# setting django environment
+from django.core.wsgi import get_wsgi_application
+sys.path.append("/websites_dir/django_spoken/spoken")
+os.environ["DJANGO_SETTINGS_MODULE"] = "spoken.settings"
+application = get_wsgi_application()
+from django.core.mail import EmailMultiAlternatives
+from django.template.loader import get_template
+from django.template import Context
+from config import *
+from cms.models import *
+
+users = User.objects.filter(is_active = 0, date_joined__lte=(datetime.now()-timedelta(days=7)))
+sent = 0
+notsent = 0
+count = 0
+tot_count = len(users)
+print 'users not activated yet : ',tot_count
+subject = 'Gentle Reminder to activate the Spoken Tutorial account'
+success_log_file_head = open(LOG_ROOT+'activation_reminder_mail_success_list.txt',"w")
+fail_log_file_head = open(LOG_ROOT+'activation_reminder_mail_fail_list.txt',"w")
+for user in users:
+    message = '''
+Dear User,
+
+Thank you for registering on Spoken Tutorial website.
+This is a gentle reminder from Spoken Tutorial Team to activate your account. 
+Kindly activate your account by clicking on the activation link which has been sent to you at the time of registration.
```

Comment:
Let's put the activation link here only, so user do not have search his/her email again.